src/content/structured/get-started/custom-theme.mdx

```diff
@@ -39,9 +44,144 @@ Alternatively, you can include the `ic-theme` component and set the `color` prop
+<ic-alert heading="Inherits from system/browser settings" variant="info"></ic-alert>`,
+      long: `.theme-container {
+    border: "1px solid var(--ic-architectural-white)",
+    padding: "var(--ic-space-sm)"
+  }
+</style>
+<body>
+  <div>
+    {shortCode}
+  </div>
+  <script>
+    function handleClick() {
+      theme = document.querySelector("ic-theme");
+      theme.theme = theme.theme === "light" ? "dark" : "light";
+    }
+  </script>`,
+    },
+  },
+  {
+    technology: "React",
+    snippets: {
+      short: `<IcButton onClick={handleClick}>Update ic-theme</IcButton>
+<IcTheme theme={theme} className={classes.themeContainer}>
+  <IcAlert heading="Set at component level" variant="info" theme="light" />
+  <IcAlert heading="Inherits from ic-theme" variant="info" />
+</IcTheme>
+<IcAlert heading="Inherits from system/browser settings" variant="info" />`,
+      long: [
+        {
+          language: "Typescript",
+          snippet: `const [theme, setTheme] = useState<"light" | "dark">("light");
+const handleClick = () => {
+  setTheme(theme === "light" ? "dark" : "light");
+};
+const useStyles = createUseStyles({
+  themeContainer: {
+    border: "1px solid var(--ic-architectural-white)",
+    padding: "var(--ic-space-sm)"
+  },
+});
+const classes = useStyles();
+<>
+  {shortCode}
+</>`,
+        },
+        {
+          language: "Javascript",
+          snippet: `const [theme, setTheme] = useState("light");
+const handleClick = () => {
+  setTheme(theme === "light" ? "dark" : "light");
+};
+const useStyles = createUseStyles({
+  themeContainer: {
+    border: "1px solid var(--ic-architectural-white)",
+    padding: "var(--ic-space-sm)"
+  },
+});
+const classes = useStyles();
+<>
+  {shortCode}
+</>`,
+        },
+      ],
+    },
+  },
+];
+
+export const DarkModeExample = () => {
+  const [theme, setTheme] = useState("light");
+  const handleClick = () => {
+    setTheme(theme === "light" ? "dark" : "light");
+  };
+  return (
+    <>
+      <IcButton onClick={handleClick}>Update ic-theme</IcButton>
+      <IcTheme
+        theme={theme}
+        style={{
+          border: "1px solid var(--ic-architectural-white)",
+          padding: "var(--ic-space-sm)",
+        }}
+      >
+        <IcAlert
+          heading="Set at component level"
+          variant="info"
+          theme="light"
+        />
+        <IcAlert heading="Inherits from ic-theme" variant="info" />
+      </IcTheme>
+      <IcAlert heading="Inherits from system/browser settings" variant="info" />
+    </>
+  );
+};
+
+<ComponentPreview
+  style={{
+    flexDirection: "column",
+    gap: "var(--ic-space-sm)",
+    background: "#333333",
+  }}
+  snippets={snippetsDefault}
+>
+  <DarkModeExample />
+</ComponentPreview>
+
+## Monochrome
+
+The `monochrome` prop can be set to `true` to remove the colour from components to help them display on certain backgrounds. It will display as black and white whilst adjusting to the dark mode inheritance mentioned above.
```

Comment:
I'm wondering if there should be some description on how the colours of monochrome change, between light and dark mode?

Reply:
Isn't it different for each component? Think the individual component examples should cover this

Reply:
Yeah it does, but I'm not sure if this makes it clear, that as you switch between dark and light mode, the actual colour will change here (even if we don't specify what it is). I wasn't sure if the last sentence was trying to say that, but I wasn't clear, when I knew it